2025-03-27 12:17:38,375:DEBUG:certbot._internal.main:certbot version: 2.10.0
2025-03-27 12:17:38,375:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/bin/certbot
2025-03-27 12:17:38,375:DEBUG:certbot._internal.main:Arguments: ['-v', '--webroot', '--agree-tos', '--renew-by-default', '--preferred-challenges', 'http-01', '--email', 'vaxelaire.yohem@gmail.com', '--webroot-path', '/var/www/certbot', '-d', 'n8n.vaxelaire.fr', '-n']
2025-03-27 12:17:38,375:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-03-27 12:17:38,435:DEBUG:certbot._internal.log:Root logging level set at 20
2025-03-27 12:17:38,436:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2025-03-27 12:17:38,437:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7fe9881ffec0>
Prep: True
2025-03-27 12:17:38,437:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7fe9881ffec0> and installer None
2025-03-27 12:17:38,437:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2025-03-27 12:17:38,500:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2243553505', new_authzr_uri=None, terms_of_service=None), b674a225a17192af95a621d2ec1bc036, Meta(creation_dt=datetime.datetime(2025, 2, 22, 10, 55, 38, tzinfo=<UTC>), creation_host='faa0276bdd8e', register_to_eff=None))>
2025-03-27 12:17:38,501:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2025-03-27 12:17:38,503:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2025-03-27 12:17:38,901:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 1042
2025-03-27 12:17:38,902:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 27 Mar 2025 12:17:38 GMT
Content-Type: application/json
Content-Length: 1042
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "profiles": {
      "classic": "https://letsencrypt.org/docs/profiles#classic",
      "shortlived": "https://letsencrypt.org/docs/profiles#shortlived (not yet generally available)",
      "tlsserver": "https://letsencrypt.org/docs/profiles#tlsserver (not yet generally available)"
    },
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.5-February-24-2025.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert",
  "v32OtITu1nw": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
}
2025-03-27 12:17:38,921:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for n8n.vaxelaire.fr
2025-03-27 12:17:38,923:DEBUG:acme.client:Requesting fresh nonce
2025-03-27 12:17:38,923:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2025-03-27 12:17:39,054:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-03-27 12:17:39,054:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 27 Mar 2025 12:17:38 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: Gsl22hvlfv5VGesYUoUfoDEf4656wcR3mI6mCrh13OIZqW0oqrY
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-03-27 12:17:39,054:DEBUG:acme.client:Storing nonce: Gsl22hvlfv5VGesYUoUfoDEf4656wcR3mI6mCrh13OIZqW0oqrY
2025-03-27 12:17:39,055:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "n8n.vaxelaire.fr"\n    }\n  ]\n}'
2025-03-27 12:17:39,057:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJHc2wyMmh2bGZ2NVZHZXNZVW9VZm9ERWY0NjU2d2NSM21JNm1DcmgxM09JWnFXMG9xclkiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "Ha335OPrLjbtVQsqGhvlLie2QlgnhPTuLd8EvboNBcIRgLpUJ2rt9aou0kDC7r9CE_Jm5cEMWZUlU11po1r-ZxhIIfW3ukVHfJbON-ceKQnjCXL-6sgw4jsObNZbnt-A-kSL1nouyHME8xkMD54jhO1k5GJjjOCwMrWH_mhOd6BGytfASk5jJEJ_UGLQiai4q2z-Ypj-wo7r9Lm4GUaGAT2rnCOlJcp2BUXFkP64Qof992tMDY_VarbMmlatdTZp1_Sq8mRhYQvWywn0E0jBdt_GiGqGk4HM0giX7cU1FQekiAer_dij7WcnTI8J-iSeyrMIUnM7tyisOmE8fBRY7w",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm44bi52YXhlbGFpcmUuZnIiCiAgICB9CiAgXQp9"
}
2025-03-27 12:17:39,253:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 350
2025-03-27 12:17:39,254:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Thu, 27 Mar 2025 12:17:39 GMT
Content-Type: application/json
Content-Length: 350
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2243553505/367912780116
Replay-Nonce: 3u7v6Bm_yBVC3DsSel3AgiGrOm2zDUHMOpi3Xb-9MErtLI2N-6Q
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-04-03T12:17:39Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "n8n.vaxelaire.fr"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/496234284506"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2243553505/367912780116"
}
2025-03-27 12:17:39,254:DEBUG:acme.client:Storing nonce: 3u7v6Bm_yBVC3DsSel3AgiGrOm2zDUHMOpi3Xb-9MErtLI2N-6Q
2025-03-27 12:17:39,254:DEBUG:acme.client:JWS payload:
b''
2025-03-27 12:17:39,256:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/496234284506:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICIzdTd2NkJtX3lCVkMzRHNTZWwzQWdpR3JPbTJ6RFVITU9waTNYYi05TUVydExJMk4tNlEiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyNDM1NTM1MDUvNDk2MjM0Mjg0NTA2In0",
  "signature": "cievuXrTEwYOGLvSnvFq4cJlHax9ba6SgFX_E5Q6wb10j_Xhs6hP3fnf5KoZbtgGiA2nQn2eLqMrnQPqfKm813DttkGf-oWtZYYbcUtwt0SE-f7MZsMCoWi0fRImI4IQjSntR32Rh49yYA3MvdvDySU78QNWirGX14ib9skFiYRZQqZfWtJxHjXEfZlf8_7u4EyxrZyxy-VqJLyGZaIntAOkTzOPse6MwSuf56_ifshge61qFKCc_f_pNZhBX4KX0raHsREU0vvr6SMIQ3z4U8kio0QilUf9dZOecEp0xSFlqnSwsgvlh78K06uSvcJhYgUsre_i3QCY_KtWewUTnQ",
  "payload": ""
}
2025-03-27 12:17:39,405:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2243553505/496234284506 HTTP/1.1" 200 824
2025-03-27 12:17:39,405:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 27 Mar 2025 12:17:39 GMT
Content-Type: application/json
Content-Length: 824
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: Gsl22hvlpgRAo1nfMaIbtBP5sjy4qZM1SyuSZygC7DXy6eTGlBs
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "n8n.vaxelaire.fr"
  },
  "status": "pending",
  "expires": "2025-04-03T12:17:39Z",
  "challenges": [
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/496234284506/qKhXhw",
      "status": "pending",
      "token": "9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns"
    },
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/496234284506/CH3Myg",
      "status": "pending",
      "token": "9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns"
    },
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/496234284506/jW66Kw",
      "status": "pending",
      "token": "9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns"
    }
  ]
}
2025-03-27 12:17:39,405:DEBUG:acme.client:Storing nonce: Gsl22hvlpgRAo1nfMaIbtBP5sjy4qZM1SyuSZygC7DXy6eTGlBs
2025-03-27 12:17:39,406:INFO:certbot._internal.auth_handler:Performing the following challenges:
2025-03-27 12:17:39,406:INFO:certbot._internal.auth_handler:http-01 challenge for n8n.vaxelaire.fr
2025-03-27 12:17:39,407:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2025-03-27 12:17:39,407:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2025-03-27 12:17:39,408:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns
2025-03-27 12:17:39,408:DEBUG:acme.client:JWS payload:
b'{}'
2025-03-27 12:17:39,410:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/496234284506/jW66Kw:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJHc2wyMmh2bHBnUkFvMW5mTWFJYnRCUDVzank0cVpNMVN5dVNaeWdDN0RYeTZlVEdsQnMiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzIyNDM1NTM1MDUvNDk2MjM0Mjg0NTA2L2pXNjZLdyJ9",
  "signature": "Ck1aY0l6saRj5YZhKLKafxwk-ECvHlnavSrL8rZ9-nL-OtDVyrwAXo8m1-GfECvv9OE9HMp26W2UU0P7wim4qdnsU2VDbKcBDgp0TFNLfg_xRI5vYLXO4OZoZu4heW4YqUfyNrxZBMwWePREUIbR0w_ej26i6fcgKd3FBFNEAsCblTJItYJRu-8rBRT92YYGwO5c6lROdafy3JUb3R75jSmTyqVSE-vZusJ26RcH6NIrXhiEYpk0v7rRHpmdCQcTUaG1Oef_iWVLDnSj9MTEWOSxjS8MtLV7jbFD8tVywekPVldlkrB5toAk2GGoawZReTIN-Q22pRYYPO7X8BcUQg",
  "payload": "e30"
}
2025-03-27 12:17:39,550:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall/2243553505/496234284506/jW66Kw HTTP/1.1" 200 195
2025-03-27 12:17:39,551:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 27 Mar 2025 12:17:39 GMT
Content-Type: application/json
Content-Length: 195
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/496234284506>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/496234284506/jW66Kw
Replay-Nonce: Gsl22hvl_qIEYq0NHfdUMTi6J0HVGktMOrmh22Hdy07jzjWx_-4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/496234284506/jW66Kw",
  "status": "pending",
  "token": "9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns"
}
2025-03-27 12:17:39,551:DEBUG:acme.client:Storing nonce: Gsl22hvl_qIEYq0NHfdUMTi6J0HVGktMOrmh22Hdy07jzjWx_-4
2025-03-27 12:17:39,551:INFO:certbot._internal.auth_handler:Waiting for verification...
2025-03-27 12:17:40,552:DEBUG:acme.client:JWS payload:
b''
2025-03-27 12:17:40,553:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2243553505/496234284506:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjI0MzU1MzUwNSIsICJub25jZSI6ICJHc2wyMmh2bF9xSUVZcTBOSGZkVU1UaTZKMEhWR2t0TU9ybWgyMkhkeTA3anpqV3hfLTQiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIyNDM1NTM1MDUvNDk2MjM0Mjg0NTA2In0",
  "signature": "jOfQ7s4KEb1bXBPI-jIFiZWg8drxOyRhPCrSRFGEEun6vIfclL3_5WewZHIfAwcHEi1uxcWmrzqO0Htra-_ASfYcinwoEs-3uKebyYzVhAjwhjjYKFcKiCABYfrh5I8k7j7EMXlVNGTKgZxaJnAzl6x_3ayzl_sEdh2WRZhk6ksWJerDBIiMMByUl1NqPMWXyNeIbB27AIjNu8_dTB8QoK-v6rSpKXQMgXXt-DKZHd9x04Xz2QPhwXeMlOwInhy2d80zvNiiZ-2QL9Y-IfR_9FfAljFrHql8qcnxdG0KeEq-2IamQz28tDb5oE6UmD2tTqXXF3KOaXm_vMeeReehRQ",
  "payload": ""
}
2025-03-27 12:17:40,691:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2243553505/496234284506 HTTP/1.1" 200 1447
2025-03-27 12:17:40,692:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Thu, 27 Mar 2025 12:17:40 GMT
Content-Type: application/json
Content-Length: 1447
Connection: keep-alive
Boulder-Requester: 2243553505
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: Gsl22hvl4BsrAoXKD-1cCJiIISsTdrg4X1cky3YpSmt654gFhog
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "n8n.vaxelaire.fr"
  },
  "status": "invalid",
  "expires": "2025-04-03T12:17:39Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2243553505/496234284506/jW66Kw",
      "status": "invalid",
      "validated": "2025-03-27T12:17:39Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "91.108.113.88: Fetching http://n8n.vaxelaire.fr/.well-known/acme-challenge/9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns: Error getting validation data",
        "status": 400
      },
      "token": "9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns",
      "validationRecord": [
        {
          "url": "http://n8n.vaxelaire.fr/.well-known/acme-challenge/9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns",
          "hostname": "n8n.vaxelaire.fr",
          "port": "80",
          "addressesResolved": [
            "91.108.113.88",
            "2a02:4780:28:d15a::1"
          ],
          "addressUsed": "2a02:4780:28:d15a::1"
        },
        {
          "url": "http://n8n.vaxelaire.fr/.well-known/acme-challenge/9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns",
          "hostname": "n8n.vaxelaire.fr",
          "port": "80",
          "addressesResolved": [
            "91.108.113.88",
            "2a02:4780:28:d15a::1"
          ],
          "addressUsed": "91.108.113.88"
        }
      ]
    }
  ]
}
2025-03-27 12:17:40,692:DEBUG:acme.client:Storing nonce: Gsl22hvl4BsrAoXKD-1cCJiIISsTdrg4X1cky3YpSmt654gFhog
2025-03-27 12:17:40,692:INFO:certbot._internal.auth_handler:Challenge failed for domain n8n.vaxelaire.fr
2025-03-27 12:17:40,693:INFO:certbot._internal.auth_handler:http-01 challenge for n8n.vaxelaire.fr
2025-03-27 12:17:40,693:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: n8n.vaxelaire.fr
  Type:   connection
  Detail: 91.108.113.88: Fetching http://n8n.vaxelaire.fr/.well-known/acme-challenge/9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns: Error getting validation data

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2025-03-27 12:17:40,694:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2025-03-27 12:17:40,694:DEBUG:certbot._internal.error_handler:Calling registered functions
2025-03-27 12:17:40,694:INFO:certbot._internal.auth_handler:Cleaning up challenges
2025-03-27 12:17:40,694:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/9fookUjcuNn3fZQYj4dKavABs7ehzzkFoo-if2ifFns
2025-03-27 12:17:40,695:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2025-03-27 12:17:40,695:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 1894, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/usr/lib/python3.12/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2025-03-27 12:17:40,698:ERROR:certbot._internal.log:Some challenges have failed.
